src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -177,6 +186,48 @@ protected int submit(List<O> ops) throws InterruptRunException, IllegalStateExce
         return submit(ops, 0, ops.size());
     }
 
+    
+	private boolean invokePathRead(final PathOperation<? extends PathItem> pathOp) {
+		final int READER_TIMEOUT_MS = 100;
+		StreamManager streamManager = StreamManager.create(controllerURI);
```

Comment:
It may be useful to use "final" where possible

Reply:
Good question. Ideally we should reuse the stream manager instance in the same way as scopes and streams. E.g. we should have a concurrent dictionary of the stream managers mapped by the controller URI as keys.